README.md

```diff
@@ -29,6 +29,7 @@ To make it work, you need to provide these **required parameters**:
 
 And you can provide some other *optional parameters*:
 - `appMountId`: The `<div>` element id on which you plan to mount a JavaScript app.
+- `appMountHtmlSnippet`: A small snippet of HTML that will be inserted in the `<div>` element the appMountId is attached to. 
```

Comment:
I would've slightly preferred a shorter name for these options, e.g., `Html` instead of `HtmlSnippet`, but they'll be great anyway, thanks.